turbo.json

```diff
@@ -63,7 +63,7 @@
       "outputs": []
     },
     "lint": {
-      "dependsOn": [],
+      "dependsOn": ["build"],
```

Comment:
Several lint rules depend on types being available for local packages, this ensures they are. Going forward, we might want to split linting into type-aware and not type-aware to optimize this (cc @nikosdouvlis).

Reply:
Spot on Bryce. Expanding a little on the above, the general idea is that we want cheap jobs to run and possibly fail early, before other more expensive operations like package building, unit and integration tests etc.